kiwi/bootloader/install/grub2.py

```diff
@@ -42,12 +42,22 @@ def post_init(self, custom_args):
                 'root device node name required for grub2 installation'
             )
 
+        self.grub2_target = 'i386-pc'
+        self.grub2_install_dev = self.device
         self.root_mount = MountManager(
             custom_args['root_device']
         )
         self.boot_mount = MountManager(
             custom_args['boot_device']
         )
+        self.modules =' '.join(Defaults.get_grub_bios_modules())
+        self.grub2_install_args = ['--skip-fs-probe']
+        if 'prep_device' in custom_args:
```

Comment:
the prep_device is required for ppc iirc, it would be nice if this is checked in post_init early as it's done for the other devices